### PR TITLE
721 update readme file

### LINKED ISSUE #721: Update the README file
## Brief description:
After changing the installation method to Dev Containers, we won't need a detailed description on how to install our projects through the scripts. This way, the README should be updated accordingly. Let's try to keep it minimal and easy to follow. 
## How to achieve it:
Edit the README file in the project root directory. Here is a [cheat sheet](https://www.markdownguide.org/cheat-sheet/) for markdown

### PR DESCRIPTION
**Pull request by Isak-Bego**
#721 Updated the README.md file to reflect changes in the project setup and structure

## New project setup workflow:
Since we changed to installing our project locally using Dev Containers. The README is updated accordingly to follow the steps need to accomplish this task. Moreover, I added a section for the WSL users who might face issues with Docker along the way. Following the instructions should lead to a successful attempt to running the project on a Dev Container. 

resolves #721

## How to verify:

- [ ] Follow the steps on How to set up the project

- [ ] Check if you could successfully set it up using those instructions

### CODE REVIEW

**Review by IvanObreshkov — COMMENTED** [review]
This is from me if @asynchroza has something to add. Otherwise good job, I love how you have put links to the different docs.

File: README.md [hunk 1]
```diff
@@ -7,67 +7,82 @@
...
-cd ~/go/src/monolith && make post-wsl
+>After this step you should wait until the container is built. It will install all the dependencies needed for development on its own.
 
-# make sure to check the output towards the end as certain installations may require manual intervention.
-```
+### 6. Check if Dev Container is running successfully
+If your Dev Container is running successfully you should be able to see the following in the bottom-right of your
+Visual Studio Code client.
+
+<img src="https://i.ibb.co/qnYQ6Qp/Screenshot-2024-10-07-at-2-33-55-PM.png" alt="image" border="0">
```
Comment (asynchroza) [inline README.md:80]: What I would suggest that you start doing is upload all images in a `/docs` directory in the repository and point to them with a relative path.

`image info`

File: README.md [hunk 2]
```diff
@@ -7,67 +7,82 @@
...
 
-### Easy access to repo
 
-```bash
-alias spa="cd ~/go/src/monolith"
-```
 ---
 ### __HOW TO RUN PROJECT__ or so called __GUM__:
```
Comment (asynchroza) [inline README.md:84]: Rename to `How to run project`

File: README.md [hunk 3]
```diff
@@ -79,136 +94,21 @@ Spin up local server instances:
...
-Similarly to the Makefile phonies, the three different options change the point towards which api requests are being made.
-```
-local api -> localhost:8000 / :6969
-prod api -> https://thehub-aubg.com
-dev api -> https://dev.thehub-aubg.com
-```
-
 **For deployments**:
 
 <img src="https://i.ibb.co/7nBqHkn/image.png" alt="image" border="0">
```
[the remark this answers is not in the document]
Comment (asynchroza) [reply README.md:100]: Is this the virtual machine thing? All deployment related stuff should totally be removed from the cli imo. The only thing the cli should handle is starting the services from the root folder of the project.

**Comment by asynchroza** [comment]
Unrelated to the changes, I would usually add a hashtag before the issue number so that when it gets merged it links to the actual issue

File: README.md (hunk 2 above)
Comment (Isak-Bego) [reply README.md]: Fixed in a401561

File: README.md (hunk 1 above)
Comment (Isak-Bego) [reply README.md]: Fixed in a401561

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Isak-Bego) [reply README.md]: Fixed in a401561

**Review by asynchroza — APPROVED** [review]
LGTM

**Pull request merged**